tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -153,6 +188,10 @@ void UILabel::ConvertStaticMappings(NIBWriter* writer, XIBObject* obj) {
         AddInt(writer, "UINumberOfLines", _numberOfLines);
     }
 
+    if (_lineBreakMode != 4) {
```

Comment:
will file one before push.

---
In reply to: [96538840](https://github.com/Microsoft/WinObjC/pull/1701#discussion_r96538840) [](ancestors = 96538840,96535041,96054766)